client/src/components/RequestModal.js

```diff
@@ -101,11 +101,26 @@ export default class RequestModal extends Component {
 
     submitRequest(e) {
         e.preventDefault()
+        
         this.setState({
             submitting: true
         }, async () => {
             try {
-                const result = true;
+                const jwtVal = document.cookie.replace(/(?:(?:^|.*;\s*)jwt\s*\=\s*([^;]*).*$)|^.*$/, "$1");
+                const parsedJWT = JSON.parse(atob(jwtVal.split('.')[1]));
+                const requesterId = parsedJWT.id
+                let timeOffset = new Date().getTimezoneOffset()
+                timeOffset = -((timeOffset/60)*100)
+                const payload = {
+                    studentId: requesterId,
```

Comment:
nit: Can we use `requestor`/`requestee` or `mentee`/`mentee` to accommodate the fact that alumni-alumni requests will also be possible?